tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/decisiontree/ProbabilisticClassifier.scala

```diff
@@ -39,6 +38,20 @@ object ProbabilisticClassifier {
   implicit object ProbabilisticClassifierJsonFormat
       extends RootJsonFormat[ProbabilisticClassifier] {
 
+    // Override the Spray/JSON serialization for maps with integer keys, because these
```

Comment:
These should go either in the `DecisionTree` object, or in a new `JsonProtocol` top-level object.